js/api-docs-redirect.js

```diff
@@ -5,6 +5,11 @@
       case '#authentication': return '/developers/api/authentication'
       case '#api-keys':
       case '#key-restrictions': return `/developers/api/authentication${hash}`
+      case '#api-specification': return '/developers/api/api-specification'
+      case '#call-an-algorithm':
+      case '#input-output':
+      case '#query-parameters':
+      case '#error-handling': return `/developers/api/api-specification${hash}`
       default: return `/developers/api${hash}`
```

Comment:
Same endless loop possibility here as I mentioned in the other PR.

Reply:
Fixed!